hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -15,8 +15,7 @@ filesets:
       - lowrisc:prim:clock_inv
       - lowrisc:prim:lc_dec
       - lowrisc:prim:lfsr
-      - lowrisc:ip:pinmux_reg
-      - lowrisc:ip:pinmux_component
+      - lowrisc:ip_interfaces:pinmux_reg
```

Comment:
Looks like ast is missing deps here. It needs pinmux_pkg.sv, which was previously part of the pinmix_component core. ast probably can just depend on earlgrey's generated cores, since it's an earlgrey-specific IP..

This is likely the source of the AscentLint failure.

Reply:
Thanks, very good spotting! Fixed.